package.json

```diff
@@ -21,7 +21,7 @@
     "ganache": "ganache-cli -p 8545 --gasLimit 0xfffffffffff --time '2017-05-10T00:00:00+00:00' -s 0",
     "publish": "lerna publish",
     "test": "yarn clean && yarn compile && lerna run test --concurrency=1",
-    "check_dep_graph": "source venv/bin/activate && pip3 install -r requirements.txt && python scripts/dependency_checker.py"
+    "deps": "source venv/bin/activate && pip3 install -r requirements.txt && python3 scripts/dependency_checker.py"
```

Comment:
make sure you update circle command.

Reply:
on second thought this doesn't make much of a difference since python is defaulting to python3 anyways in the venv. Explicitness is still good though.